url.bs

```diff
@@ -280,10 +280,14 @@ U+005C (\), or U+005D (]).
 <a>domain</a> <var>domain</var>, runs these steps:
 
 <ol>
- <li><p>Let <var>result</var> be the result of running <a abstract-op lt=ToASCII>Unicode ToASCII</a> with
- <i>domain_name</i> set to <var>domain</var>, <i>UseSTD3ASCIIRules</i> set to false,
- <i>processing_option</i> set to <i>Nontransitional_Processing</i>, and <i>VerifyDnsLength</i> set
- to false.
+ <li>
+  <p>Let <var>result</var> be the result of running <a abstract-op lt=ToASCII>Unicode ToASCII</a>
+  with <i>domain_name</i> set to <var>domain</var>, <i>UseSTD3ASCIIRules</i> set to false,
+  <i>CheckHyphens</i> set to false, <i>processing_option</i> set to
+  <i>Nontransitional_Processing</i>, and <i>VerifyDnsLength</i> set to false.
```

Comment:
Yeah. I guess I could add a pointer to #110 in the XXX comment about them. Although CheckBidi seems uncontroversial.

Basically I'm not getting feedback from browsers on this which is rather frustrating.